scripts/prepare_dutchf3.py

```diff
@@ -105,66 +132,90 @@ def split_patch_train_val(data_dir, stride, patch, per_val=0.2, log_config=None)
     logger.info("Splitting data into patches .... ")
     logger.info(f"Reading data from {data_dir}")
 
-    labels_path = _get_labels_path(data_dir)
-    logger.info(f"Loading {labels_path}")
-    labels = np.load(labels_path)
+    # labels_path = _get_labels_path(data_dir)
+    logger.info(f"Loading {label_file}")
+    labels = np.load(label_file)
     logger.debug(f"Data shape [iline|xline|depth] {labels.shape}")
 
     iline, xline, depth = labels.shape
     # Inline sections
-    train_iline_range, test_iline_range = _get_aline_range(iline, per_val)
+    try:
+        train_iline_range, test_iline_range = _get_aline_range(iline,
+                                                               per_val,
+                                                               slice_steps)
+    except RuntimeError as re:
+        logger.error(re)
 
     # Xline sections
-    train_xline_range, test_xline_range = _get_aline_range(xline, per_val)
+    try:
+        train_xline_range, test_xline_range = _get_aline_range(xline,
+                                                               per_val,
+                                                               slice_steps)
+    except RuntimeError as re:
+        logger.error(re)
 
     # Generate patches from sections
     # Process inlines
-    horz_locations = range(0, xline - stride, stride)
-    vert_locations = range(0, depth - stride, stride)
+    horz_locations = range(0, xline - patch_size + 1, stride)
+    vert_locations = range(0, depth - patch_size + 1, stride)
```

Comment:
How does this change is related to  skip_steps?  Should it be part of a separate PR?

Reply:
This was resolved in @kirasoderstrom PR and I incorporated here so I don't have conflicts once it's merged to the MSFT DeepSeismic repo.